test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -80,11 +80,58 @@ public class ScanConsistencyIT extends AccumuloClusterHarness {
 
   private static final Logger log = LoggerFactory.getLogger(ScanConsistencyIT.class);
 
-  @SuppressFBWarnings(value = {"PREDICTABLE_RANDOM", "DMI_RANDOM_USED_ONLY_ONCE"},
-      justification = "predictable random is ok for testing")
+  public static void main(String[] args) {
+    /**
+     * @formatter:off
```

Comment:
This is a javadoc. It doesn't make sense to turn off formatting for javadocs... because the end result is still rendered HTML, not formatted plain text. Instead, use html tags, like `<p>`, `<ul>`, etc. If you do it right, the javadoc will be formatted reasonably so that it's readable.

Also, if you want `<` and `>` to render properly, like you have near the bottom of this javadoc, you're going to need to express it as `&lt;` or `&gt;`.

Reply:
Thanks, I updated the comment